README.md

```diff
@@ -1,222 +1,74 @@
+* [Layout](https://github.com/mobify/stencil-utils/tree/master/dist/layout)
+* [Spacing](https://github.com/mobify/stencil-utils/tree/master/dist/spacing)
+* [Text](https://github.com/mobify/stencil-utils/tree/master/dist/text)
+* [Visibility](https://github.com/mobify/stencil-utils/tree/master/dist/visibility)
 
 
 ## Requirements
 
-Stencil is written in Sass (SCSS syntax), and requires Sass 3.4. It also relies on [Spline](http://github.com/mobify/spline) and works best when included with [Vellum](http://github.com/mobify/vellum).
-
+- [Sass](http://sass-lang.com/) 3.4+
+- [Grunt](http://gruntjs.com/)
+- Adaptive.js 2.1 +
 
 ## Installation
 
-Install with [Bower](http://bower.io):
+Install with [Grunt](http://gruntjs.com/):
 
 ```
-bower install mobify-stencil --save
-```
-
-Once installed, the Stencil library lives in your project’s `bower_components` directory. Individual components will provide a stylesheet partial (Sass) including example markup and (optionally) JavaScript.
-
-Use the Sass `@import` directive to include a component’s styles. For example, in `my-project/styles/stylesheet.scss`, I can import the Grid component like so:
-
-```scss
-@import '../bower_components/mobify-stencil/dist/components/grid/grid';
+grunt component:install:utils
 ```
 
-Always import Stencil components *after* your Sass variables and *before* any of your own components. This ensures that Stencil receives the values you want for any of its configurable variables while allowing you to reliably build on what it provides.
-
+Once installed, the Stencil Utilities live in your project’s `app/bower_components` directory. Individual utilities will provide a stylesheet, including example markup and (optionally) JavaScript. To learn more, visit the [dev centre documentaton](/).
 
-## Using a Stencil component
 
-Stencil components are like any component you build yourself, except (as with all dependecies) you should never modify them directly. You can *configure* a Stencil component’s CSS by overriding its configurable variables in your own stylesheets. You can also *extend* components by styling them as you see fit and creating new variations on existing components. Feel free to override Stencil styles as well, although Stencil tries to be as minimal as possible so you shouldn’t often need to.
+## Using a Stencil Utility
 
-One of Stencil’s advantages is that you can work more with HTML and write less custom CSS. Once your theming is in place and you’ve build any custom components you need, you might find you can compose whole screens of UI by simply writing HTML implementing the markup patterns (structure and classes) defined by the component.
+Utility classes are conceptually a bit like a Sass mixin; they allow you to add new style rules to an existing component, except that they are applied as classes in your markup. An example utility class is `.u-width-full`. It does exactly what it says: make an element expand to fill the full width of its container.
```

Comment:
When it was decided that utilities gonna have `u-` prefix? I remember starting this thread but it was not supported https://github.com/mobify/stencil/issues/141